src/js/components/Accordion/stories/Custom.js

```diff
@@ -3,9 +3,17 @@ import { storiesOf } from '@storybook/react';
 
 import { SubtractCircle, AddCircle } from 'grommet-icons';
 
-import { Accordion, AccordionPanel, Box, Grommet, Text } from 'grommet';
+import {
+  grommet,
+  Accordion,
+  AccordionPanel,
+  Box,
+  Grommet,
+  Text,
+} from 'grommet';
+import { deepMerge } from 'grommet/utils';
 
-const CustomAccordionTheme = {
+const CustomAccordionTheme = deepMerge(grommet, {
```

Comment:
Why do we need to deepMerge with grommet? Can't we just create our theme object as JSON?

Also, can we rename the variable `customTheme`, as upper case sounds more like a component.

Reply:
>Why do we need to deepMerge with grommet? Can't we just create our theme object as JSON?

I like the font style of 'grommet', but I can add the font directly to the theme instead.

> Also, can we rename the variable customTheme, as upper case sounds more like a component.

agreed.